bench/db_bench.ml

```diff
@@ -468,6 +497,9 @@ let run input json_file_name data_dir metrics_flag =
         [ `OverWrite; `All; `Index ],
         "OverWrite" );
       ((fun () -> lmdb_benchmarks ()), [ `Lmdb; `All ], "Run lmdb benchmarks");
+      ( (fun () -> Index.merge 450_000 "write_random"),
+        [ `Merge; `All; `Index ],
+        "Force merge of \"write_random\" with 450_000/500_000 entries in log" );
```

Comment:
Even more important than the number of entries in the log, it would be useful to know the number of entries in the whole index, since the merge operation depends on the entries in `data` more than the ones in `log`.

Reply:
It is printed at the beginning of each benchmark in `init`, so I think it is redundant to print it here too. I can add it if you want.

Reply:
I see, it's fine then!